### DISCUSSION TITLE
Game creation

### DISCUSSION THREAD

**Discussion by Rostence** [community/community#60457] [Mobile] [answered]
Who can help me in my game creation I want team to build big business videos game

**Comment by farisei77** [comment]
Hmm...i interesting in casino online and.. many years old I were closed betatester in popular now command game as LOL (Leage of Legend)

**Reply by Rostence** [reply]
So what can we do about casino games now

**Reply by farisei77** [reply]
Maybe creat integration casino thems in casual game

**Reply by Rostence** [reply]
Yap I think you are making since so who can we start about it

**Reply by farisei77** [reply]
Hmmm... only things and ideas

**Reply by Rostence** [reply]
Ok I will start app now

**Comment by ossanna16** [comment, marked as answer]
Hi @Rostence ,
Thanks for being a part of the GitHub Community! We're glad you're here.

I wanted to jump in and mention that if you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on building games. It's definitely possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
